src/Layers/FeatureLayer/FeatureManager.js

```diff
@@ -64,6 +64,9 @@ export var FeatureManager = VirtualGrid.extend({
    */
 
   onAdd: function (map) {
+    // include 'Powered by Esri' in map attribution
+    Util.setEsriAttribution(map);
```

Comment:
Shouldn't this line read `setEsriAttribution(map)` only, as you're explicitly importing `setEsriAttribution`?